swaglyrics/__main__.py

```diff
@@ -39,7 +39,7 @@ def unsupported_precheck(force: bool = False) -> None:
     print('Updating unsupported.txt from server.')
     with open(unsupported_txt, 'w', encoding='utf-8') as f:
         try:
-            unsupported_songs = requests.get(f'{backend_url}/master_unsupported')
+            unsupported_songs = requests.get(f'{backend_url}/master_unsupported', timeout=10)
```

Comment:
I would recommend keeping the timeout value as a constant in a config file or something similar that we are using.

Since it is a value used at multiple places, that would be way better.

Reply:
I can do it, but I was wondering if readability would be hampered because then you'd need to open the constants file instead of just having it right there? Also it wouldn't necessarily reduce verbosity because it's a matter of `timeout=10` vs `timeout=API_TIMEOUT`.

It would be useful if someone plans to change the value though, but I'm not sure if that will happen.

Reply:
It is not about verbosity but as you yourself pointed out, changing it becomes easy and it is a very good programming principle.

Reply:
I'll do it then!